src/components/markdown/Markdowns.js

```diff
@@ -5,6 +5,7 @@ import Markdown from './Markdown';
 function Markdowns({
   markdowns
```

Comment:
Do you need to pass the whole markdown object, or just the title?

Reply:
We have to map through the array of markdowns, so we need the entire array.